test/test.js

```diff
@@ -48,6 +48,19 @@ $.mockjax( {
 	}
 } );
 
+$.mockjax( {
```

Comment:
looks like this mocking request alters global state. will it affect other tests?